UI/Web/src/app/sidenav/_components/side-nav-item/side-nav-item.component.scss

```diff
@@ -22,21 +22,21 @@
     align-items: center;
     display: flex;
     justify-content: space-between;
-    padding: 0 0 0 10px;
     width: 100%;
     height: auto;
-    min-height: 40px;
+    min-height: 50px;
     overflow: hidden;
     cursor: pointer;
-    font-size: 0.9rem;
+    font-size: 1.05rem;
+    font-family: sans-serif;
```

Comment:
Can you explain why this change?

Reply:
The font size needed to be increased to match the increase of the icon size for the font family I was just testing if it looked better but after removing it now I realized it looks better without the change so I agree with that being reverted